xml/operating_system.xml

```diff
@@ -28,7 +28,7 @@
 
   <fingerprint pattern="^(?i:(?:Microsoft )?Windows 10 Mobile(?:\s([a-z]+))?(?: Edition)?)$">
     <description>Windows 10 Mobile</description>
-    <example os.product="Windows 10 Mobile">Windows 10 Mobile Edition</example>
+    <example os.product="Windows 10 Mobile" os.edition="Edition">Windows 10 Mobile Edition</example>
```

Comment:
This looks buggy--probs my should be "Mobile," but it's a current shipping defect. This patch only makes it obvious since it interpolates the field value from the match groups derived from the example string.